node-admin/src/main/java/com/yahoo/vespa/hosted/node/admin/maintenance/coredump/CoredumpHandler.java

```diff
@@ -154,29 +158,40 @@ Optional<ContainerPath> getCoredumpToProcess(ContainerPath containerCrashPath, C
      *
      * @return path to directory inside processing directory which contains the enqueued core dump file
      */
-    Optional<ContainerPath> enqueueCoredump(ContainerPath containerCrashPath, ContainerPath containerProcessingPath) {
+    Optional<ContainerPath> enqueueCoredump(NodeAgentContext context, ContainerPath containerCrashPath, ContainerPath containerProcessingPath) {
+        Predicate<String> isCoreDump = filename -> !HS_ERR_PATTERN.matcher(filename).matches();
```

Comment:
FYI there's `Pattern::asMatchPredicate` that does this

Reply:
Nice to know, but it didn't improve the readability.